report-deployment-status-to-slack/action.yml

```diff
@@ -56,32 +60,65 @@ runs:
       id: setup
       shell: bash
       run: |
-        case "${{ inputs.event }}" in
-          start)
-            SLACK_MESSAGE=":loading: Updating ${{ inputs.environment }} to \`${{ inputs.tag }}\` on ${{ inputs.project }}.
-            ${{ inputs.url }}
-            SHA: \`${{ inputs.commit-sha }}\`"
-            ;;
+        if [[ "${{ inputs.db_migration_detected }}" ]]; then
```

Comment:
@Eeebru Can you point me to where we are checking if the branch is a feature branch (ie. not `main`, `rc`, or `hotfix-rc`)?

Reply:
This is our pretty generic `deploy-to slack` workflow, so that check doesn't exist here. I looked into the [QA-deploy ](https://github.com/bitwarden/devops/blob/eb1dd152c04d84f67f6cdf94c81ea0e7295ce815/.github/workflows/QA-deploy.yml)workflow file, I don't think we are doing that check. So essentially, that DB migration warning will reflect on any branch that has Migration detected, including prod branches

Reply:
Gotcha. We would like to only alert when the branch that is being deployed is a feature branch as mentioned in the Jira task. 

We want to create awareness for only the DB migrations that are on feature branches and not for all DB migrations. Alerting on DB migrations in `main`, `rc`, and `hotfix-rc` will create false positives and not have the overall effect of creating the awareness for the very specific behavior that we want to change.

Reply:
Hello Joseph, I added the caveat [here](https://github.com/bitwarden/devops/blob/7674e15401ac845a1fb56caeadcaefa085174b35/.github/workflows/QA-deploy.yml#L73)